### PR TITLE
Add input

### PR DESCRIPTION
**Pull request by inductionotg**

### CODE REVIEW

File: components/input/input.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    <link rel="stylesheet" href="../../styles/documentation.css">
+    <link rel="stylesheet" href="input.css">
+    <script src="https://kit.fontawesome.com/bf13f25644.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/default.min.css">
+    <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/highlight.min.js"></script>
+
+</head>
+
+<body>
+    <div class="pagewrapper">
```
Comment (tejas-muthya37) [inline components/input/input.html:18]: Try to maintain a naming convention like camelCase or try to name your classes like this -> 'page-wrapper' or 'page_wrapper'

File: components/input/input.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+                        <i class="fa fa-twitter footer-icon fa-lg" aria-hidden="true"></i>
+                    </a>
+                </li>
+                <li class="footer-item">
+                    <a href="https://www.linkedin.com/in/ritesh-sinha-25822a146/" target="_blank">
+                        <i class="fa fa-linkedin footer-icon fa-lg" aria-hidden="true"></i>
+                    </a>
+                </li>
+            </ul>
+            <p class="copywright">
```
Comment (tejas-muthya37) [inline components/input/input.html:150]: I think you meant 'copyright'. The typo might lead to a bit of confusion as to why you have named it this way.

**Review by tejas-muthya37 — APPROVED** [review]
Rest of the code looks good to me. Great work :) Good luck.

**Comment by anandsr21harshit** [comment]
Tejas has given good advise. Follow a proper naming convention.

**Review by anandsr21harshit — APPROVED** [review]

**Pull request merged**